index.js

```diff
@@ -54,19 +53,19 @@ async function doStuff() {
             core.info('Ignoring scheduled tweets for now.');
             return;
         }
-        const twitterClient = new Twitter({
-            consumer_key: core.getInput('twitterConsumerKey'),
-            consumer_secret: core.getInput('twitterConsumerSecret'),
-            access_token_key: core.getInput('twitterAccessTokenKey'),
-            access_token_secret: core.getInput('twitterAccessTokenSecret'),
+        const twitterClient = new TwitterApi({
+            appKey: core.getInput('twitterConsumerKey'),
+            appSecret: core.getInput('twitterConsumerSecret'),
+            accessToken: core.getInput('twitterAccessTokenKey'),
+            accessSecret: core.getInput('twitterAccessTokenSecret'),
```

Comment:
Didn't rename the inputs due to compatibility, but happy to do so if you are willing to update your inputs ;)

Reply:
If these new names match what twitter has in their dev console better I'd say the input names shoukd also be updated.

Reply:
![image](https://user-images.githubusercontent.com/330324/165601244-754275ed-231c-4d6c-9b13-3523a6fdbf07.png)

I'd say this could go in either direction if we look at the dev console 😁